packages/wrangler/src/utils/print-bindings.ts

```diff
@@ -760,12 +770,13 @@ export function printBindings(
 		logger.log();
 	}
 
-	if (hasConnectionStatus) {
-		logger.once.info(
+	if (hasConnectionStatus && !isConnectedStatusExplained) {
+		log(
 			dim(
 				`\nService bindings, Durable Object bindings, and Tail consumers connect to other Wrangler or Vite dev processes running locally, with their connection status indicated by ${chalk.green("[connected]")} or ${chalk.red("[not connected]")}. For more details, refer to https://developers.cloudflare.com/workers/runtime-apis/bindings/service-bindings/#local-development\n`
```

Comment:
```suggestion
				`\nService bindings, Durable Object bindings, and Tail consumers connect to other Wrangler or Vite processes running locally, with their connection status indicated by ${chalk.green("[connected]")} or ${chalk.red("[not connected]")}. For more details, refer to https://developers.cloudflare.com/workers/runtime-apis/bindings/service-bindings/#local-development\n`
```

Removing `dev` as it could also be `preview`.

Reply:
I will update it in the follow up PR with all other message changes (as they impact a huge amount of snapshot tests in wrangler).